internal/worker/server.go

```diff
@@ -134,6 +138,15 @@ func notFoundHandler(writer http.ResponseWriter, request *http.Request) {
 	jsonErrorf(writer, http.StatusNotFound, "not found")
 }
 
+func (s *Server) StatusHandler(writer http.ResponseWriter, request *http.Request, _ httprouter.Params) {
```

Comment:
```suggestion
func (s *Server) statusHandler(writer http.ResponseWriter, request *http.Request, _ httprouter.Params) {
```

Please, can you change it to a private method for consistency?